tests/test_advanced.py

```diff
@@ -12,11 +15,36 @@
     sat_64_repo,
     sat_65_repo,
 )
-
 from testfm.decorators import capsule, stubbed
 from testfm.log import logger
 
 
+@pytest.fixture(scope='function')
+def setup_sync_plan(request, ansible_module):
+    sync_plan_name = gen_string('alpha')
+    sync_date = datetime.datetime.today().strftime('%Y-%m-%d')
+    setup = ansible_module.command(
+        'hammer sync-plan create --name {0} --enabled true \
+        --interval "weekly" --sync-date {1} --organization-id 1'.format(
+            sync_plan_name, sync_date))
+    for result in setup.values():
+        assert result["rc"] == 0
+    setup = ansible_module.command(
+        'hammer sync-plan info --name {0} --organization-id 1'.format(
```

Comment:
Have you checked with json or csv format of hammer to avoid splitting [int(s) for s in sync_id.split() ?